plugins/de.ovgu.featureide.fm.core/src/de/ovgu/featureide/fm/core/base/IFeatureStructure.java

```diff
@@ -134,4 +134,6 @@
 	void setRelevantConstraints();
 
 	void setRelevantConstraints(List<IConstraint> constraints); // Marcus, if calculated outside the class, see FeatureUtils.setRelevantConstraints(...)
+
+	void upDateInherited(FeatureAttribute attribute);
```

Comment:
I think it should be named updateInheritedFeatureAttribute or something similar.